Pirate/Program.cs

```diff
@@ -0,0 +1,190 @@
+﻿using System;
+
+namespace Pirate
+{
+    class Program
+    {
+
+        static string GetUserResponse(string question)
+        {
+            Console.WriteLine(question);
+            string userResponse = Console.ReadLine();
+            while (userResponse == "")
+            {
+                Console.WriteLine("Please enter yee a response.");
+                Console.WriteLine(question);
+                userResponse = Console.ReadLine();
+            }
+            return userResponse;
+        }
+
+
+        static void Main(string[] args)
+        {
+
+
+            Console.WriteLine("ARRRR Welcome YEE to my Pirate Adventure!");
+            Console.WriteLine("");
+            string name = GetUserResponse("AAR what is your PIRATE name?");
+            Console.Clear();
+            string lastname = GetUserResponse("And what is yee Pirate last name?");
+            Console.Clear();
+            string newpiratename = (name + " " + lastname);
+
+            Console.WriteLine($"Welcome {newpiratename}, To my pirate adventure!");
+
+            string answer;
+
+            do
+
```

Comment:
Consider removing this blank line that separates the keyword `do` from the start of the scope